general/server/media/movies.md

```diff
@@ -50,3 +49,14 @@ If labels are not added to the end of filenames, as shown above, each file will
 
 > [!Note]
 > To group media manually, long-click or right-click media to highlight then select additional media to merge. Use the new bar that appears to 'Group Versions'.
+
+### Order of Versions
+
+Movie versions are presented in an alphabetically sorted list. An exception applies to resolution names, which are sorted in descending order from highest to lowest resolution. A version name qualifies as a resolution name when ending with either a `p` or an `i`.
+
+> [!Note]
+> The first movie version in the list is the one selected by default.
+
+**Examples of sorting**
```

Comment:
```suggestion
#### Examples of sorting
```
If I read the "MarkdownLinter" right thenn this is what it expects you to use :thinking:

Reply:
That's how I'd understand it as well.

Reply:
Thanks.